governance/votes/Executive vote - July 11, 2024.md

```diff
@@ -0,0 +1,125 @@
+
+If this executive proposal passes, a payment of **219,125 DAI** will be made to a smart contract controlled by Aave Governance at [0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c](https://etherscan.io/address/0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c) as part of the Aave-SparkLend Revenue Share.
+
+### Update Chainlink Keeper Network Treasury Address
+
+- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/amend-keeper-network-chainlink-automation-v2-1/24593/3)
+- **Proposal**: [Forum post](https://forum.makerdao.com/t/amend-keeper-network-chainlink-automation-v2-1/24593)
+
+If this executive proposal passes, the Chainlink Keeper Network Treasury Address will be updated to [0xBE1cE564574377Acb17C2b7628E4F6dd38067a55](https://etherscan.io/address/0xBE1cE564574377Acb17C2b7628E4F6dd38067a55). This is a requirement to facilitate the upgrade of Chainlink Automation for MakerDAO from v2.0 to v2.1.
+
+The required contract call is:
+
+`keeperRegistry20.migrateUpkeeps([54617870736745152906494510799239970615562458165653206109311709851388548258865], keeperRegistry21);`
+
+### Stability Scope Parameter Updates
+
+- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/stability-scope-parameter-changes-14/24594/2)
+- **Proposal**: [Forum post](https://forum.makerdao.com/t/stability-scope-parameter-changes-14/24594)
+
+If this executive proposal passes, the following parameter changes will be made.
+
+#### [Stability Fee](https://mips.makerdao.com/mips/details/MIP104#14-3-1-3-stability-fee-sf-) Reductions
+
+- ETH-A: Decrease by 1 percentage point, from 8.25% to **7.25%**.
+- ETH-B: Decrease by 1 percentage point, from 8.75% to **7.75%**.
+- ETH-C: Decrease by 1 percentage point, from 8% to **7%**.
+- WSTETH-A: Decrease by 1 percentage point, from 9.25% to **8.25%**.
+- WSTETH-B: Decrease by 1 percentage point, from 9% to **8%**.
+- WBTC-A: Decrease by 1 percentage point, from 9.75% to **8.75%**.
+- WBTC-B: Decrease by 1 percentage point, from 10.25% to **9.25%**.
+- WBTC-C: Decrease by 1 percentage point, from 9.5% to **8.5%**.
+
+#### [Dai Savings Rate (DSR)](https://mips.makerdao.com/mips/details/MIP104#3-2-the-dai-savings-rate) Reduction
+
+The DSR will decrease by 1 percentage point, from 8% to **7%**.
+
+### Delegate Compensation
+
+**Authorization**: [Atlas 2.6 Aligned Delegates](https://mips.makerdao.com/mips/details/MIP101#2-6-aligned-delegates-ads-gov6)  
+**Proposal**: [Forum post](https://forum.makerdao.com/t/june-2024-aligned-delegate-compensation/24617)
+
+If this executive proposal passes a total of **209.17 MKR** will be distributed for Aligned Delegate Compensation as follows. This is the first month where Aligned Delegates may request payments to multiple addresses. If a payout is sent to a different recipient at an Aligned Delegate's request, the Aligned Delegate's name will appear in parentheses for transparency. 
```

Comment:
```suggestion
If this executive proposal passes a total of **209.17 MKR** will be distributed for Aligned Delegate Compensation as detailed below.
```
I'd suggest to change the language because it's usually expected that the detail follows directly after the "as follows" statement.